### ISSUE TITLE
SSLHandshakeException on android 4.4

### ISSUE THREAD

**Issue by ligi** [walleth/walleth#353]
```
javax.net.ssl.SSLHandshakeException: javax.net.ssl.SSLProtocolException: SSL handshake aborted: ssl=0xb84ec510: Failure in SSL library, usually a protocol error
error:14077410:SSL routines:SSL23_GET_SERVER_HELLO:sslv3 alert handshake failure (external/openssl/ssl/s23_clnt.c:741 0x8d91d990:0x00000000)
```

**Comment by ligi** [comment]
might be related: https://stackoverflow.com/questions/37006416/sslexception-connection-closed-by-peer-on-android-4-x-versions/37021685#37021685

**Comment by ligi** [comment]
looks like this is the solution: https://github.com/square/okhttp/issues/2372#issuecomment-244807676
